sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_constants.py

```diff
@@ -32,10 +32,6 @@
 _IS_ON_APP_SERVICE = "WEBSITE_SITE_NAME" in environ
 # TODO: Add environment variable to enabled diagnostics off of App Service
 _IS_DIAGNOSTICS_ENABLED = _IS_ON_APP_SERVICE
-# TODO: Enabled when duplicate logging issue is solved
```

Comment:
What did this refer to again?

Reply:
Basically when we used the old "enable" method on the exporter logger, it caused some duplication. For many reasons, such as our namespace change and message ids, it is no longer relevant.